lib/python/picongpu/input/parameters.py

```diff
@@ -0,0 +1,103 @@
+"""
+This file is part of PIConGPU.
+
+Copyright 2017 PIConGPU contributors
+Authors: Sebastian Starke, Jeffrey Kelling
+License: GPLv3+
+"""
+
+from numpy import log10
+
+
+class Parameter(object):
+    def __init__(self, name, ptype, unit, default, value=None):
```

Comment:
please provide doc strings for the class and methods.

See for example `lib/python/picongpu/plugins/energy_histogram.py`